### PR TITLE
Changed tonality

### PR DESCRIPTION
**Pull request by DylanMcBean**
I reworded some sentences to sound less passive and more engaging for the reader.

### CODE REVIEW

**Comment by BrendanChambersBourgeois** [comment]
👍

**Comment by WathikAhmed** [comment]
Looking good!

File: README.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ The 4PrivacyEngine has been a work in progress since late 2020 - with the specif
 
 **Initial advisory team code review.** Refactor code based on feedback from advisors.
 
-**Final advisory team whitepaper review.**  Use final advisor feedback to prepare whitepaper for public review.
+**Final advisory team whitepaper review.**  Use final advisor feedback to prepare a whitepaper for public review.
 
 **Second advisory team code review.**  Refactor code based on feedback from advisors in preparation for additional external code review and testing.
 
-**Limited external code review.** Make code available to limited external organizations for testing and review.  This is the final step before putting code into public repository.
+**Limited external code review.** Make code available to limited external organizations for testing and review. The review is the final step before putting code into a public repository.
```
Comment (martimalek) [inline README.md:18]: Honestly I'd change back `The review` to `This`. There are multiple reviews going on and the reader may not be sure which one is being referred to. Not a native english speaker though so not sure at all... the other changes look great!!

**Comment by TechnikEmpire** [comment]
Just a tip here guys, people are watching for PR's and commits of like, actual things, so maybe we can not have 80 people debating which words to use.

**Comment by DylanMcBean** [comment]
I don't see an issue with making this sound friendlier for people to read, as nothing is getting added until next year.

**Pull request closed without merging**